src/simulate/assets/rigid_body.py

```diff
@@ -128,3 +128,26 @@ def __post_init__(self):
 
     def __hash__(self) -> int:
         return id(self)
+
+    def copy(self) -> "RigidBodyComponent":
+        """
+        Create a copy of this rigid body with the same values.
+
+        Returns:
+            copy (`RigidBodyComponent`):
+                A copy of this rigid body
+        """
+        instance_copy = RigidBodyComponent(
```

Comment:
Same here, probably better to loop over the fields and convert/copy/raise an error

Reply:
Fixed